srcalc/src/main/java/gov/va/med/srcalc/web/controller/AdminHomeController.java

```diff
@@ -37,6 +50,19 @@ public AdminHomeController(final AdminService adminService)
     @RequestMapping(method = RequestMethod.GET)
     public String defaultPage(final Model model)
     {
+        final ImmutableCollection<RiskModel>  riskModels = fAdminService.getAllRiskModels();
+
+        // Use a set since there are multiple entries in the risk models list for each model.
+        //
+        Set<EditRiskModel> editRMs = new TreeSet<EditRiskModel>(  );
+
+//        fLogger.debug( "There are {} Risk Models.", riskModels.size());
```

Comment:
I don't understand why you keep commenting out logging statements. One of the advantages of using a logging library is that disabled logging statements (e.g., the statement level is DEBUG but the configured level is INFO) have insignificant performance impact.